### ISSUE TITLE
Extensions Crash after a while

### ISSUE THREAD

**Issue by Daviid-P** [microsoft/vscode#170836]
Does this issue occur when all extensions are disabled?: No

# TL;DR
I think this error makes my extensions crash
```
2023-01-09 11:04:26.967 [error] Blocked vscode-file request vscode-file://vscode-app/c:/Users/David/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/editor/common/services/editorSimpleWorker.nls.js
2023-01-09 11:04:27.191 [info] Extension host with pid 70708 exited with code: 0, signal: null.
```

--------

- VS Code Version: 1.74.2
- OS Version: Windows_NT x64 10.0.22621

Steps to Reproduce:

It's hard to reproduce, I'm not sure what's causing the crash and sometimes it's quite a few minutes before it happens, I'm Bisecting right now but I'm not sure I'll get anywhere.

Here's the error code in OUTPUT:
```
2023-01-09 10:50:35.657 [info] Extension host terminating: renderer closed the socket
2023-01-09 10:50:35.668 [error] An error occurred when deactivating the subscriptions for extension 'bmewburn.vscode-intelephense-client':
2023-01-09 10:50:35.668 [error] TypeError: Cannot read private member from an object whose class did not declare it
	at Fe (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:6644)
	at Object.dispose (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:92:12956)
	at y (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:10:12121)
	at d.X (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:11533)
	at c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9301
	at Array.map (<anonymous>)
	at d.Q (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9288)
	at d.terminate (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9563)
	at f.terminate (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:110:1397)
	at b (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:120:17815)
	at Socket.<anonymous> (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:120:15524)
	at Socket.emit (node:events:538:35)
	at Pipe.<anonymous> (node:net:687:12)
2023-01-09 10:50:35.671 [error] Error: Client is not running and can't be stopped. It's current state is: starting
	at b.shutdown (c:\Users\David\.vscode\extensions\bmewburn.vscode-intelephense-client-1.9.2\lib\extension.js:1:139825)
	at b.stop (c:\Users\David\.vscode\extensions\bmewburn.vscode-intelephense-client-1.9.2\lib\extension.js:1:139406)
	at b.stop (c:\Users\David\.vscode\extensions\bmewburn.vscode-intelephense-client-1.9.2\lib\extension.js:1:299360)
	at t.deactivate (c:\Users\David\.vscode\extensions\bmewburn.vscode-intelephense-client-1.9.2\lib\extension.js:1:66792)
	at d.X (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:11333)
	at c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9301
	at Array.map (<anonymous>)
	at d.Q (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9288)
	at d.terminate (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9563)
	at f.terminate (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:110:1397)
	at b (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:120:17815)
	at Socket.<anonymous> (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:120:15524)
	at Socket.emit (node:events:538:35)
	at Pipe.<anonymous> (node:net:687:12)
2023-01-09 10:50:35.681 [info] Extension host with pid 102616 exiting with code 0
```

Then after a while the extensions get started again on their own.

I don't think it's the extension `bmewburn.vscode-intelephense-client` because if I deactivate it I end up getting the same error with a different extension name.

Not sure if related but trying to find a way to replicate I ended up with:

- Modify a file and save.
- Go to Source Control.
- Crash with error message on OUTPUT > Main `2023-01-09 10:56:58.919 [error] Blocked vscode-file request vscode-file://vscode-app/c:/Users/David/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/editor/common/services/editorSimpleWorker.nls.js
`

This is with `bmewburn.vscode-intelephense-client` disabled, however I no longer see the big stacktrace, only:

```
2023-01-09 11:00:31.584 [info] Extension host terminating: renderer closed the socket
2023-01-09 11:00:31.619 [info] Extension host with pid 69456 exiting with code 0
2023-01-09 11:01:32.234 [info] Extension host with pid 114972 started
```

I tried with ALL extensions enabled and got a bigger stacktrace:

```
2023-01-09 11:04:26.697 [info] Extension host terminating: renderer closed the socket
2023-01-09 11:04:26.714 [error] An error occurred when deactivating the subscriptions for extension 'bmewburn.vscode-intelephense-client':
2023-01-09 11:04:26.715 [error] TypeError: Cannot read private member from an object whose class did not declare it
	at Fe (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:6644)
	at Object.dispose (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:92:12956)
	at y (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:10:12121)
	at d.X (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:11533)
	at c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9301
	at Array.map (<anonymous>)
	at d.Q (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9288)
	at d.terminate (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9563)
	at f.terminate (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:110:1397)
	at b (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:120:17815)
	at Socket.<anonymous> (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:120:15524)
	at Socket.emit (node:events:538:35)
	at Pipe.<anonymous> (node:net:687:12)
2023-01-09 11:04:26.717 [error] An error occurred when deactivating the subscriptions for extension 'Gruntfuggly.todo-tree':
2023-01-09 11:04:26.718 [error] AggregateError: Encountered errors while disposing of store
	at y (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:10:12197)
	at d.X (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:11533)
	at c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9301
	at Array.map (<anonymous>)
	at d.Q (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9288)
	at d.terminate (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9563)
	at f.terminate (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:110:1397)
	at b (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:120:17815)
	at Socket.<anonymous> (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:120:15524)
	at Socket.emit (node:events:538:35)
	at Pipe.<anonymous> (node:net:687:12)
2023-01-09 11:04:26.718 [error] An error occurred when deactivating the subscriptions for extension 'jebbs.plantuml':
2023-01-09 11:04:26.719 [error] TypeError: Cannot read properties of undefined (reading 'dispose')
	at UI.dispose (c:\Users\David\.vscode\extensions\jebbs.plantuml-2.17.5\out\src\ui\ui.js:24:21)
	at c:\Users\David\.vscode\extensions\jebbs.plantuml-2.17.5\out\src\providers\previewer.js:38:87
	at Array.map (<anonymous>)
	at Previewer.dispose (c:\Users\David\.vscode\extensions\jebbs.plantuml-2.17.5\out\src\providers\previewer.js:38:76)
	at y (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:10:12121)
	at d.X (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:11533)
	at c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9301
	at Array.map (<anonymous>)
	at d.Q (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9288)
	at d.terminate (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:106:9563)
	at f.terminate (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:110:1397)
	at b (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:120:17815)
	at Socket.<anonymous> (c:\Users\David\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:120:15524)
	at Socket.emit (node:events:538:35)
	at Pipe.<anonymous> (node:net:687:12)
2023-01-09 11:04:26.735 [info] Extension host with pid 85544 exiting with code 0
```

And also the same `Blocked vscode-file request` error:
```
2023-01-09 11:04:26.967 [error] Blocked vscode-file request vscode-file://vscode-app/c:/Users/David/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/editor/common/services/editorSimpleWorker.nls.js
2023-01-09 11:04:27.191 [info] Extension host with pid 70708 exited with code: 0, signal: null.
```

**Comment by gjsjohnmurray** [comment]
Maybe try the `Start Extension Bisect` command in an attempt to pinpoint which of your extensions is causing this.

**Comment by Daviid-P** [comment]
> Maybe try the `Start Extension Bisect` command in an attempt to pinpoint which of your extensions is causing this.

I did this, got to the end (1 extension disabled), I did the steps
- Modify a file and save.
- Go to Source Control.

And it crashed, I clicked "This is bad" and it said that it couldn't identify the extension.

**Comment by lszomoru** [comment]
@Daviid-P, are you able to reproduce the issue if you start code using the [--disable-extensions](https://code.visualstudio.com/docs/editor/command-line#_working-with-extensions) flag?

**Comment by Daviid-P** [comment]
@lszomoru Pretty sure I can not.
